### PR TITLE
Zenghui Zou's Matplotlib and Seaborn homework (reviewed)

### PR DESCRIPTION
**Pull request by ZenghuiZou**

### CODE REVIEW

**Comment by ZenghuiZou** [comment]
@iphysresearch  Zenghui Zou's Matplotlib and Seaborn homework

File: 2023/homework/Zou_Zenghui/homework-matplotlib_Zou.ipynb
Comment (iphysresearch) [inline 2023/homework/Zou_Zenghui/homework-matplotlib_Zou.ipynb:1]: 完成Matplotlib数据可视化作业！

File: 2023/homework/Zou_Zenghui/homework-seaborn_Zou.ipynb
Comment (iphysresearch) [inline 2023/homework/Zou_Zenghui/homework-seaborn_Zou.ipynb:1]: 完成Seaborn数据可视化作业！

**Review by iphysresearch — APPROVED** [review]
@ZenghuiZou 
恭喜完成数据可视化的基础和扩展作业！

**Pull request merged**